sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -116,6 +119,7 @@ public partial class BlobServiceClient
         public BlobServiceClient(System.Uri serviceUri, Azure.Storage.Blobs.BlobClientOptions options = null) { }
         public BlobServiceClient(System.Uri serviceUri, Azure.Storage.StorageSharedKeyCredential credential, Azure.Storage.Blobs.BlobClientOptions options = null) { }
         public string AccountName { get { throw null; } }
+        public bool CanGenerateSasUri { get { throw null; } }
```

Comment:
We were also going to add a `CanCallGetUserDelegationKey` property to indicate when we had a `TokenCredential` and customers could at least make a valid request with `GetUserDelegationKey` if they wanted one of those to generate a SAS.  Should that happen as part of this change?  Or should we file another work item?

Reply:
This name needs to match the method and that appears to have been rename to `GenerateAccountSasUri`.

Reply:
I can make a separate work item for that. https://github.com/Azure/azure-sdk-for-net/issues/16470